### PR TITLE
Upgrade Jepsen to 0.3.1

### PR DESCRIPTION
**Pull request by yito88**
There are many changes, but, it should be compatible.
Steady tests worked well.
After this upgrade, I'll refine the nemesis injections.

Ref. https://github.com/jepsen-io/jepsen/releases/tag/v0.3.0

Also, It updated dependencies in `project.clj`.

### CODE REVIEW

File: cassandra/project.clj [hunk 1]
```diff
@@ -3,9 +3,12 @@
...
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
-  :dependencies [[org.clojure/clojure "1.10.1"]
+  :dependencies [[org.clojure/clojure "1.11.1"]
                  [org.clojure/java.jmx "0.3.1"]
-                 [jepsen "0.2.1"]
+                 [jepsen "0.3.1" :exclusions [net.java.dev.jna/jna
+                                              net.java.dev.jna/jna-platform]]
+                 [net.java.dev.jna/jna "5.11.0"]
+                 [net.java.dev.jna/jna-platform "5.11.0"]
```
Comment (yito88) [inline cassandra/project.clj:11]: For aarch64

File: scalardl/project.clj [hunk 2]
```diff
@@ -13,8 +17,7 @@
                    :plugins [[lein-cloverage "1.1.2"]]}
              :use-released {:dependencies [[com.scalar-labs/scalardl-java-client-sdk "4.0.0-SNAPSHOT"
                                             :exclusions [org.slf4j/slf4j-log4j12
-                                                         com.oracle.database.jdbc/ojdbc8
-                                                         software.amazon.awssdk/*]]]}
+                                                         com.scalar-labs/scalardb]]]}
```
Comment (yito88) [inline scalardl/project.clj:20]: for local `use-released` test

File: scalardl/src/scalardl/core.clj [hunk 3]
```diff
@@ -114,7 +114,7 @@
   []
   (retry-when-exception (fn [package]
                           (c/su (debian/install package)))
-                        [[:openjdk-8-jre]]
+                        [[:openjdk-11-jre]]
```
Comment (yito88) [inline scalardl/src/scalardl/core.clj:117]: for Scalar DL

File: .circleci/config.yml [hunk 4]
```diff
@@ -4,7 +4,7 @@ jobs:
     working_directory: ~/scalar-jepsen
 
     docker:
-      - image: circleci/clojure:openjdk-8-lein-2.9.1
+      - image: circleci/clojure:openjdk-11-lein-2.9.8
```
Comment (brfrn169) [inline .circleci/config.yml:7]: Does Jepsen 0.3.1 only supports `jdk 11` or later?

**Review by brfrn169 — APPROVED** [review]
Left one question, but LGTM. Thank you!

File: .circleci/config.yml (hunk 4 above)
Comment (brfrn169) [reply .circleci/config.yml:7]: Ah, I just took a look at this:
https://github.com/scalar-labs/verification/pull/55

I see Jepsen requires Java 11 or later. Thank you.

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**